datasets/za/wanted/crawler.py

```diff
@@ -49,7 +49,7 @@ def crawl_person(context: Context, cell: html.HtmlElement):
     name, crime, status = map(str.strip, match.groups())
 
     # either first or last name is considered a bare minimum to emit a person entity
-    unknown_spellings = ["Unknown", "Uknown"]
+    unknown_spellings = ["Unknown", "Uknown", "unknown"]
```

Comment:
On a slack about other data from my country

![image](https://github.com/user-attachments/assets/8c9139b1-39ab-4846-bfba-184b88653c71)

We seem to have a knack for this :D

Reply:
indeed `- "Unknnown m" - "Unkniwn kg"` :))